app/views/devise/sessions/new.html.slim

```diff
@@ -19,7 +16,7 @@ p.my3 == t('notices.log_in_consent.text', app: APP_NAME, link: link)
 .clearfix.pt1.border-top
   - if @sp_name && @sp_return_url
```

Comment:
yeah I thought about that...I think it would make sense to go that route if we returned a partial from that method that included all of the code in this `if` statement, since actually returning the erb/HTML from a decorator would be annoying (need to pull in ActionView::Helper stuff)

If we go route of returning a partial, we'd want to return an empty partial from `SessionDecorator`. In micropurchase, we had a shared partial named `null` that we used for this. Should we do that here or wait until a further iteration?